algorithm_distillation/sentiment-data/ppo_config.yml

```diff
@@ -0,0 +1,49 @@
+model:
+  model_path: "lvwerra/gpt2-imdb"  # Name of hf model to load
+  tokenizer_path: "gpt2"  # Name of hf tokenizer to load
+  model_type: "AcceleratePPOModel"  # Name of accelerate model type to load
+  num_layers_unfrozen: 2  # Number of bottom layers to freeze during training
+
+train:
+  seq_length: 48  # Size of LM context
+  epochs: 100 # Train for max(epochs, total_steps)
+  total_steps: 10000  # Train for max(epochs, total_steps)
+  batch_size: 128  # batch size
+
+  lr_init: 1.0e-4  # init learning rate
+  lr_target: 1.0e-4  # target final learning rate
+  opt_betas: [0.9, 0.95] # adam betas
+  opt_eps: 1.0e-8  # adam eps
+  weight_decay: 1.0e-6  # weight decay param
+
+  checkpoint_interval: 10000  # checkpoint interval
+  eval_interval: 100  # eval interval
+
+  pipeline: "PromptPipeline"  # prompt pipeline to load
+  orchestrator: "PPOOrchestrator"  # orchestrator to load
+
+  rollout_logging_dir: "~/Algorithm-Distillation-RLHF/algorithm_distillation/sentiment-data/rollouts"
```

Comment:
This line was causing some trouble for me. Is there a way to use relative dir instead of absolute?
Tried and didn't work. Maybe it's a problem of trlx, like we should use `pathlib.Path` instead of `os.path` here
https://github.com/CarperAI/trlx/blob/1a3461d592a567409e11e3c47b5b355bf219449f/trlx/model/accelerate_ppo_model.py#L115